ray-operator/controllers/ray/common/pod.go

```diff
@@ -493,6 +493,9 @@ func BuildPod(ctx context.Context, podTemplateSpec corev1.PodTemplateSpec, rayNo
 		initLivenessAndReadinessProbe(&pod.Spec.Containers[utils.RayContainerIndex], rayNodeType, creatorCRDType)
 	}
 
+	// add downward API environment variables for Ray default node labels
```

Comment:
"node labels" is not very clear under the context of KubeRay, so I update the comment to provide more context.

```suggestion
	// Add downward API environment variables for Ray's default node labels for Ray label-based scheduling.
```